assets/styl/_base/_symbols.styl

```diff
@@ -0,0 +1,7 @@
+// Symbols
+//
+// svg symbols
+//
+// Markup: symbols.twig
+//
+// Styleguide: Basics.Symbols
```

Comment:
I would name this section (and related files) `svg`: because if we need to style symbols, we'll add styles for `svg` selectors.